src/ArbCall/ArbFunction.jl

```diff
@@ -16,6 +16,8 @@ declaration in C. Note that note all types of function declarations
 are supported, only those which are relevant for the Arb library.
 
 ```jldoctest
+julia> using Arblib
+
```

Comment:
@Joel-Dahne It should be enough to add something like
```julia
DocMeta.setdocmeta!(
    Arblib,
    :DocTestSetup,
    :(using Arblib;);
    recursive = true,
)
```
to `docs/make.jl` so that in every `jldoctest` `Arblib` is available.
See https://documenter.juliadocs.org/stable/man/doctests/#Module-level-metadata

Reply:
I'm documenting my own package at this very moment and I'm learning this as we speak ;)

Reply:
Great! I felt like there should be such a functionality but I couldn't immediately find it.